lib/message_queue/rpc_client/command.ex

```diff
@@ -14,29 +14,28 @@ defmodule MessageQueue.RPCClient.Command do
   end
 
   defp execute(payload) do
-    try do
-      with {:ok, %{"module" => module, "function" => function, "args" => args}} <-
-             Jason.decode(payload),
-           {:ok, module} <- get_module_name(module),
-           {:ok, function} <- get_function_name(function),
-           true <- module in MessageQueue.rpc_modules(),
-           true <- Code.ensure_loaded?(module),
-           true <- function_exported?(module, function, length(args)) do
-        apply(module, function, args)
-      else
-        _ -> {:error, :not_supported_method}
-      end
-    rescue
-      error ->
-        Logger.error(inspect(error))
-        {:error, :internal_error}
+    with {:ok, %{"module" => module, "function" => function, "args" => args}} <-
+           Jason.decode(payload),
+         {:ok, module} <- get_module_name(module),
+         {:ok, function} <- get_function_name(function),
+         true <- module in MessageQueue.rpc_modules(),
+         true <- Code.ensure_loaded?(module),
+         true <- function_exported?(module, function, length(args)) do
+      apply(module, function, args)
+    else
+      _ -> {:error, :not_supported_method}
     end
+  rescue
+    error ->
+      Logger.error(inspect(error))
```

Comment:
lets add some context to understand where error comes
```suggestion
      Logger.error(ctx: __MODULE__, error: inspect(error))
```